src/server/scripts/Pandaria/GateoftheSettingSun/boss_striker_gadok.cpp

```diff
@@ -0,0 +1,752 @@
+    
+    uint8 pointIdEnd;
+    Position end;
+    
+    uint8 pointIdOutside;
+    Position outside;
+};
+
+enum eTalks
+{
+    TALK_AGGRO,
+    TALK_DEATH,
+    TALK_INTRO,
+    TALK_SLAY,
+};
+
+enum eMovements
+{
+    POINT_NORTH_START       = 1,
+    POINT_SOUTH_START       = 2,
+    POINT_WEST_START        = 3,
+    POINT_EAST_START        = 4,
+
+    POINT_NORTH_END         = 5,
+    POINT_SOUTH_END         = 6,
+    POINT_WEST_END          = 7,
+    POINT_EAST_END          = 8,
+
+    POINT_NORTH_OUTSIDE     = 9,
+    POINT_SOUTH_OUTSIDE     = 10,
+    POINT_WEST_OUTSIDE      = 11,
+    POINT_EAST_OUTSIDE      = 12,
+
+    POINT_KRIKTHIK_CIRCLE   = 13,
+
+    MOV_NORTH_SOUTH         = 0,
+    MOV_SOUTH_NORTH         = 1,
+    MOV_WEST_EAST           = 2,
+    MOV_EAST_WEST           = 3
+};
+
+StrafPointStruct StrafPoints[4] =
+{
+    { POINT_NORTH_OUTSIDE, {1258.0f, 2304.644f, 438.0f, 0.0f}, POINT_NORTH_START, {1238.007f, 2304.644f, 435.0f, 0.0f}, POINT_NORTH_END, {1153.398f, 2304.578f, 435.0f, 0.0f}, POINT_SOUTH_OUTSIDE, {1133.4f, 2304.578f, 438.0f, 0.0f} }, // North -> South
+    { POINT_SOUTH_OUTSIDE, {1133.4f, 2304.578f, 438.0f, 0.0f}, POINT_SOUTH_START, {1153.398f, 2304.578f, 435.0f, 0.0f}, POINT_SOUTH_END, {1238.007f, 2304.644f, 435.0f, 0.0f}, POINT_NORTH_OUTSIDE, {1258.0f, 2304.644f, 438.0f, 0.0f} }, // South -> North
+    { POINT_WEST_OUTSIDE,  {1195.3f, 2366.941f, 438.0f, 0.0f}, POINT_WEST_START,  {1195.299f, 2348.941f, 435.0f, 0.0f}, POINT_WEST_END,  {1195.392f, 2263.441f, 435.0f, 0.0f}, POINT_EAST_OUTSIDE,  {1195.4f, 2243.441f, 438.0f, 0.0f} }, // West  -> East
+    { POINT_EAST_OUTSIDE,  {1195.4f, 2243.441f, 438.0f, 0.0f}, POINT_EAST_START,  {1195.392f, 2263.441f, 435.0f, 0.0f}, POINT_EAST_END,  {1195.299f, 2348.941f, 435.0f, 0.0f}, POINT_WEST_OUTSIDE,  {1195.3f, 2366.941f, 438.0f, 0.0f} }  // East  -> West
+};
+
+Position CenterPos = {1195.0f, 2304.0f, 438.0f};
+
+enum eSpells
+{
+    SPELL_PREY_TIME         = 106933,
+    SPELL_IMPALING_STRIKE   = 107047,
+
+    SPELL_STRAFING_RUN      = 107342,
+    SPELL_STRAFIND_RUN_DMG  = 116298,
+
+    SPELL_RIDE_VEHICLE      = 46598,
+
+    // Disruptor
+    SPELL_BOMB              = 115110
+};
+
+enum eEvents
+{
+    EVENT_PREY_TIME         = 1,
+    EVENT_IMPALING_STRIKE   = 2,
+
+    EVENT_DISRUPTOR_BOMBARD = 3,
+};
+
+enum ePhases
+{
+    PHASE_MAIN          = 1,
+    PHASE_NORTH_SOUTH   = 2,
+    PHASE_WEST_EAST     = 3
+};
+
+enum eStrafing
+{
+    STRAF_NONE    = 0,
+    STRAF_70      = 1,
+    STRAF_30      = 2
+};
+
+#define MAX_DISRUPTOR   5
+#define MAX_STRIKER     10
+#define RADIUS_CIRCLE   100.0f
+
+class boss_striker_gadok : public CreatureScript
+{
+    public:
+        boss_striker_gadok() : CreatureScript("boss_striker_gadok") {}
+
+        struct boss_striker_gadokAI : public BossAI
+        {
+            boss_striker_gadokAI(Creature* creature) : BossAI(creature, DATA_GADOK)
+            {
+                instance = creature->GetInstanceScript();

```

Comment:
remove that line at ALL, you already inherited it with value, you can't assign new